bottles/frontend/ui/library-entry.blp

```diff
@@ -69,77 +42,113 @@ template LibraryEntry : Box {
     }
 
     [overlay]
-    Image img_icon {
-      icon-name: "applications-games-symbolic";
-      halign: end;
-      valign: end;
-      margin-bottom: 10;
-      margin-end: 10;
-    }
-  }
-
-  Box {
-    spacing: 6;
-
-    Box {
-      orientation: vertical;
-      hexpand: true;
-
-      Label label_name {
-        halign: start;
-        label: _("Item name");
-
-        styles [
-          "title",
-        ]
-      }
+    Revealer revealer_run {
+      reveal-child: false;
+      transition-type: crossfade;
+      valign: center;
 
-      Label label_bottle {
-        halign: start;
-        label: _("Bottle name");
+      Box {
+        valign: center;
+        halign: center;
 
-        styles [
-          "caption",
-        ]
+        Button btn_run {
+          valign: center;
+          halign: center;
+          label: _("Launch");
+
+          styles [
+            "osd",
+            "pill",
+          ]
+        }
+
+        [overlay]
+        Button btn_launch_steam {
+          valign: center;
+          halign: center;
+          visible: false;
+          label: _("Launch with Steam");
+
+          styles [
+            "osd",
+            "pill",
+          ]
+        }
       }
     }
 
-    [end]
-    Box {
-      Button btn_launch_steam {
-        tooltip-text: _("Launch with Steam");
-        valign: center;
-        icon-name: "bottles-steam-symbolic";
-        visible: false;
-
-        styles [
-          "flat",
-        ]
-      }
+    [overlay]
+    Revealer revealer_details {
+      reveal-child: false;
+      transition-type: crossfade;
+      valign: end;
 
-      Button btn_run {
-        valign: center;
-        icon-name: "media-playback-start-symbolic";
+      Box {
+        spacing: 6;
+        halign: fill;
+        valign: end;
+        vexpand: true;
+        margin-bottom: 10;
+        margin-start: 10;
+        margin-end: 10;
 
         styles [
-          "flat",
+          "osd",
+          "toolbar",
+          "library-entry-details",
         ]
-      }
 
-      Button btn_stop {
-        visible: false;
-        halign: center;
-        icon-name: "media-playback-stop-symbolic";
-
-        styles [
-          "flat",
-        ]
+        Box {
+          orientation: vertical;
+          hexpand: true;
+          valign: center;
+
+          Label label_name {
+            halign: start;
+            label: _("Item name");
+
+            styles [
+              "title",
+            ]
+          }
+
+          Label label_bottle {
+            halign: start;
+            label: _("Bottle name");
+
+            styles [
+              "caption",
+            ]
+          }
+        }
+
+        [end]
+        Box {
+          Button btn_remove {
+            halign: center;
+            icon-name: "list-remove-symbolic";
+            tooltip-text: _("Remove");
```

Comment:
Since @tiziodcaio and @jannuary are in favor for a trash icon, perhaps we could update the tooltip text?

```suggestion
            tooltip-text: _("Remove from library");
```

Reply:
small grammar change: Remove from Library